language-server/modules/langserver-core/src/test/resources/codeaction/source/testgen/module2/services.bal

```diff
@@ -0,0 +1,74 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/websub;
+
+service<http:Service> httpService bind { port: 9090 } {
+    sayHello(endpoint caller, http:Request request) {
+    }
+}
+
+service<http:Service> httpsService bind securedListener {
+    sayHello(endpoint caller, http:Request request) {
+    }
+}
+
+service<http:WebSocketService> wsService bind { port: 9094 } {
+    onOpen(endpoint caller) {
+
```

Comment:
Shall we remove this extra line from the resource body? Also check for other places which has this extra line inside the resource body

Reply:
@NatashaWso2 it is okay to have extra lines since this is a part of the test cases.

Reply:
But in line 6 of the same test file you have not added an extra line in the resource body?